tests/test_indexer.py

```diff
@@ -327,6 +347,65 @@ def get_search_results(cls, query, expected_hit_count):
                 time.sleep(0.5)
 
 
+class BundleBuilder:
+    def __init__(self, bundle_id=None, bundle_version=None):
+        self.bundle_id = bundle_id if bundle_id else str(uuid.uuid4())
+        self.bundle_version = bundle_version if bundle_version else self._get_version()
+        self.bundle_manifest = {
+            BundleMetadata.FORMAT: BundleMetadata.FILE_FORMAT_VERSION,
+            BundleMetadata.VERSION: self.bundle_version,
+            BundleMetadata.FILES: [],
+            BundleMetadata.CREATOR_UID: "0"
+        }
+
+    def get_bundle_id(self):
+        return f'{self.bundle_id}.{self.bundle_version}'
+
+    def add_file(self, bucket_name, name, indexed, file_id):
+        # Add the existing file to the bundle manifest
+        s3 = boto3.resource("s3")
+        file_manifest_string = s3.Object(bucket_name=bucket_name,
+                                         key=f'files/{file_id}').get()['Body'].read().decode('utf-8')
+        file_manifest = json.loads(file_manifest_string, encoding="utf-8")
+        file_uuid, _, file_version = file_id.partition(".")
```

Comment:
Ahh. `file_uuid, file_version = file_id.split(".", 1)` should work, but I'll leave it to your discretion.